tools/ci_build/build.py

```diff
@@ -580,6 +580,7 @@ def convert_arg_line_to_args(self, arg_line):
     )
     # Please note in our CMakeLists.txt this is already default on. But in this file we reverse it to default OFF.
     parser.add_argument("--disable_rtti", action="store_true", help="Disable RTTI (reduces binary size)")
+    parser.add_argument("--disable_abseil", action='store_true', help="Do no use abseil and default to STL")
     parser.add_argument(
```

Comment:
For obscure things like can they specify via cmake_extra_args so we're not adding excessive amounts of new parameters to build.py